website/content/docs/agent/telemetry.mdx

```diff
@@ -249,58 +249,73 @@ Note that if servers don't restart often, then the snapshot could have grown
 significantly since the last restore happened so last restore times might not
 reflect what would happen if an agent restarts now.
 
+### License Expiration <EnterpriseAlert inline />
+
+| Metric Name                       | Description                                                      | Unit  | Type  |
+| :-------------------------------- | :--------------------------------------------------------------- | :---- | :---- |
+| `consul.system.licenseExpiration` | Number of hours until the Consul Enterprise license will expire. | hours | gauge |
+
+**Why they're important:** 
+
+This measurement indicates how many hours are left before the Consul Enterprise license expires. When the license expires some Consul Enterprise features will cease to work.
```

Comment:
It could be useful to add an example of what happens when an enterprise feature ceases to work, to get a better sense for the impact. (Assuming this isn't documented elsewhere)

Reply:
Added.